opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1061,7 +1061,8 @@ namespace Opm {
                                                                           this->well_open_times_,
                                                                           schedule_open))
                 {
-                    this->wellState()[wname].efficiency_scaling_factor = wscale;
+                    if (this->wellState().has(wname))
```

Comment:
we shouldn't be here unless we have the well so this if should be redundant?

Reply:
ah, right - will revert